.github/actions/release-action/action.yml

```diff
@@ -49,3 +57,5 @@ runs:
         artifacts: ${{ inputs.artifacts }}
         generateReleaseNotes: ${{ inputs.generateReleaseNotes }}
         tag: ${{ inputs.tag }}
+        repo: ${{ inputs.repo }}
+        token: ${{ inputs.token }}
```

Comment:
I also can't seem to find the tags back in the repo eventhough it didn't seem to have any errors durin that step. Could you run the workflow again with the changes of today?

Reply:
I manually deleted the production tags after the failed swap so it won't be confusing. That part of the workflow worked, and everything else up until Create Release.

I've run this again, and it fails with the same error. I left the tags there now.

BTW you should be able to reproduce this by just trying to run Create Release referencing a different test repo.

Reply:
Now it puts the tags on the incorrect commit, BTW:

![image](https://github.com/user-attachments/assets/d53834fc-96bd-47cf-8a26-44d719f3015f)

What I swapped out wasn't `dev`, but the issue branch, as indicated by the staging tags.

Reply:
I was finally able to figure out with some additional local testing on my own repo's that the reason the release was failing was not because of permission issues like the error message initially suggested. But rather that the release-action takes repo name and repo owner as 2 separate inputs rather then as 1 string such as `Lombiq/GitHub-Actions`.

This should be fixed in the latest version, as well as tagging the wrong commit. Please run the action again whenever you get a chance.

Reply:
Worked now, thank you.